File: cpp/exceptions/ZeroMultiplierException.cpp
#include "ZeroMultiplierException.h"

ZeroMultiplierException::ZeroMultiplierException(const QString& what)
    : KalkException(what) {}

ZeroMultiplierException* ZeroMultiplierException::clone() const {
    return new ZeroMultiplierException(*this);
}

void ZeroMultiplierException::raise() const {
    throw *this;
}

File: java/kalkException/ZeroMultiplierException.java
package kalkException;

public class ZeroMultiplierException extends RuntimeException{
	private static final long serialVersionUID = 1L;
	
	public ZeroMultiplierException() {
		super();
	}
	
	public ZeroMultiplierException(String s) {
		super(s);
	}
}
